en/reference/services-search.html

```diff
@@ -328,6 +329,20 @@ <h2 id="renderer">renderer</h2>
           bundle="the name in &lt;artifactId&gt; in pom.xml" /&gt;
 </pre>
 
+<h2 id="significance">significance</h2>
+<p>
+The significance tag can include multiple models. Their order determines the model precedence for a given language, with the last element having the highest. The models' document frequency is used to set a token's significance value based on the inverse document frequency (IDF). To enable the use of these models, the schema needs to have a rank-profile field with the <em>significance</em> element and the  <em>use-model</em> flag set to <em>true</em>.
+</p>
+
+<p>Example of significance model with multiple models. These models are either provided by <em>Vespa</em> or can be generated with the <a href="vespa-cmdline-tools.html#vespa-significance">vepsa-signficance</a> cli. </p>
```

Comment:
Add a link to https://docs.vespa.ai/en/reference/config-files.html#model which documents the semantics of the `model-id`,`url` and `path` attributes. The first sentence can be simplified to e.g. "Example with multiple model files". There is another "vepsa" typo here ;)